src/openeo_grass_gis_driver/app.py

```diff
@@ -4,14 +4,18 @@
 from flask_restful import Api
 from werkzeug.middleware.proxy_fix import ProxyFix
 
-__author__ = "Sören Gebbert"
-__copyright__ = "Copyright 2018, Sören Gebbert, mundialis"
-__maintainer__ = "Soeren Gebbert"
-__email__ = "soerengebbert@googlemail.com"
+__license__ = "Apache License, Version 2.0"
+__author__ = "Sören Gebbert, Carmen Tawalika"
+__copyright__ = "Copyright 2018-2021, Sören Gebbert, mundialis"
+__maintainer__ = "mundialis"
+
+API_VERSION = "v1.0"
```

Comment:
Does this mean the driver supports only this specific API version, and API_VERSION would need to be adjusted if `/.well-known/openeo` changes?

Reply:
I moved the definition to `/.well-known/openeo`. But yes - it means that only one API version is supported which is reflected in the code anyway. If we again need multiple deployments of different versions in the future, we need to work with releases (or branches like in the past).